src/os/control/mapmodecontrol.js

```diff
@@ -1,134 +1,140 @@
-goog.provide('os.control.MapMode');
-
-goog.require('goog.dom');
-goog.require('goog.dom.classlist');
-goog.require('ol');
-goog.require('ol.control.Control');
-goog.require('ol.css');
-goog.require('ol.events');
-goog.require('ol.events.EventType');
+goog.module('os.control.MapMode');
+goog.module.declareLegacyNamespace();
 
+const dom = goog.require('goog.dom');
+const GoogEventType = goog.require('goog.events.EventType');
+const Control = goog.require('ol.control.Control');
+const css = goog.require('ol.css');
+const events = goog.require('ol.events');
+const EventType = goog.require('ol.events.EventType');
+const dispatcher = goog.require('os.Dispatcher');
+const MapChange = goog.require('os.MapChange');
+const osActionEventType = goog.require('os.action.EventType');
+const mapInstance = goog.require('os.map.instance');
```

Comment:
Nit: we've generally preferred to destructure requires when only calling 1 or 2 functions off of them. Doesn't make any functional difference, but if we set up the jscodeshift script to do these replaces it would be a bit more consistent to just get the `getMapContainer` function.

Reply:
Updated and resolved the merge conflict.